server/src/routes/api/event.mjs

```diff
@@ -55,7 +55,7 @@ function sendEvent(req, res) {
     });
   }
 
-  events.sendEvent(ws, userId, method, result, '', fSuccess, fErr, fFatalErr);
+  events.sendEvent(ws, method, result, `${method}`, fSuccess, fErr, fFatalErr);
```

Comment:
It looks like "userId" was dropped somehow. It should be passed, I believe, looking at the signature of the sendEvent function.

Reply:
Changed the code.